File: 03/main.js
console.log(new Date().getSeconds())
setTimeout(() => {
  console.log(new Date().getSeconds());
  console.log('first timeout')
}, 1000)

const task = setTimeout(() => {
  console.log('you can not see this line');
}, 3000)
//清除定时器
clearTimeout(task)


//setInternal按时间间隔循环执行
setInterval(() => {
  console.log('internal call')
},2000)

File: 03/main.cs


var timer1 = new System.Timers.Timer();
//设置触发间隔时长
timer1.Interval = 1000;
//设置是否重复执行
timer1.AutoReset = true;
timer1.Elapsed += (sender,e)=>{
    System.Console.WriteLine("Internal call");
};
timer1.Enabled = true;

await Task.Delay(5000);
System.Console.WriteLine("you can enter any key to stop timer1");
System.Console.ReadKey();